src/templates/CareerForm/config.js

```diff
@@ -0,0 +1,86 @@
+// import metaTags from 'src/components/MetaTags/config';
+// import actionBar from 'src/components/ActionBar/config';
+// import hubspot from 'src/components/HubSpotForm/config';
+
+// export default {
+//   label: 'Career Forms',
+//   label_singular: 'Career Form',
+//   name: 'career form',
+//   folder: 'netlify/careerforms',
```

Comment:
This should match up with the netlify folder

```suggestion
   folder: 'netlify/careers',
```